config/locales/views/application/de.yml

```diff
@@ -1,4 +1,4 @@
 de:
-  address:
-    actions:
+  application:
+    gray_box_layout:
       new: Neue Adresse hinzufügen
```

Comment:
this is strange, I am sure that I deleted the folder first and not just rename it